docs/src/domains/cdn/fastly.md

```diff
@@ -1,55 +1,57 @@
 ---
-title: "Fastly"
+title: "Configure your Fastly CDN"
+sidebarTitle: "Fastly"
 weight: 1
-description: In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache. Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
+description: Learn how to configure your Fastly CDN.
+aliases:
+  - "/golive/steps/fastly.html"
 ---
 
-{{% description %}}
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
+To set up a Fastly CDN on a Grid project,
+follow the instructions on this page.
 
-A Fastly CDN is included for Dedicated projects.
-Platform.sh doesn't offer an integrated CDN on self-service Grid projects at this time,
-but it is a common choice for customers to self-configure.
+**Dedicated users**: Fastly CDNs included in Dedicated projects are set up by Platform.sh. If you need help with your Fastly CDN settings,
+[open a support ticket](https://console.platform.sh/-/users/~/tickets/open).
+Note that when you request that a new domain is added to your Fastly service,
```

Comment:
:hammer_and_wrench: I feel like it might be clearer this way, but please let me know if you think I'm wrong. This might be less clear for non-native speakers. My problem is that `request that X is done` isn't really fully "correct", but the correct version involves subjunctives that most people will find difficult to parse.

```suggestion
When you request for a new domain to be added to your Fastly service,
```